test/integration/cmds/organization/__snapshots__/taxonomy-transform.test.ts.snap

```diff
@@ -0,0 +1,345 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
do we need this snap test?

Reply:
I think it shows us that the different methods work as we expect